README.md

```diff
@@ -55,17 +55,28 @@ The GLIF CLI maps human readable names to account addresses. Whenever you pass a
 
 `glif <command> <command-args> --from testing-account`<br />
 
+To create a read-only label for an arbitrary address:<br />
```

Comment:
Maybe this could be moved to the the "wallets" section? It feels like a secondary feature, but it's the first thing introduced.